static/app/utils/events.tsx

```diff
@@ -152,7 +152,7 @@ export function getTitle(
       // (https://github.com/getsentry/sentry/pull/19794) so we need to fall
       // back to the computed title for these.
       return {
-        title: customTitle ?? (metadata.message || event.title),
+        title: customTitle ?? (metadata.message || title),
```

Comment:
i think for perf issues the title will be coming from `metadata.title`

Reply:
Hmm, Originally it was returned from event.title, was the not intended?
<img width="248" alt="image" src="https://user-images.githubusercontent.com/44422760/189405339-b80f1f94-a186-4a84-ade3-55d125f34e76.png">

Reply:
The title in this line is actually unrelated to perf issues, its just because I destructured the object on line 106.
The title form perf issue should come from `customTitle` which is set by `metadata.title`

Reply:
Grabbing from metadata now!